pkg/controller/control.go

```diff
@@ -1078,12 +1083,35 @@ func (c *Controller) metricsStart() {
 	go func() {
 		for {
 			time.Sleep(1 * time.Second)
+			c.metricsLock.Lock()
```

Comment:
Why we need `metricsStart` instead of keeping `c.metrics` only w/o `c.latestMetrics`?